Create ui-c8y-1021-73-0-added-messaging-management-as-beta-feature-8891-graft-release-cd-9003.md (#3303)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1021-73-0-added-messaging-management-as-beta-feature-8891-graft-release-cd-9003.md

---

> date: "2025-05-23"

Reviewer: ```suggestion
date: 
```

Reviewer: The date will be set automatically on deplyoement

Reviewer 2: On which deployment? I see that a newer version `1021.73.8` is already deployed on `eu-latest`. Will it set the correct date, if we merge it as empty now?

Reviewer: It will set the date of the next deployment after the merge. Alternatively, you can figure out at which date this change was actually deployed and set in manually. But this means, that users might miss the entry if they only look for recent changes.

Reviewer 2: ```suggestion
date: ""
```

> title: Added messaging management as beta feature (#8891) [GRAFT][release/cd] (#9003)

Reviewer 2: ```suggestion
title: Messaging service - monitoring and management feature available as public preview
```

> title: Messaging service - monitoring and management feature available as public preview

Reviewer: ```suggestion
title: Messaging Service monitoring and management feature available as public preview
```

> title: Messaging Service monitoring and management feature available as a Public Preview

Reviewer: ```suggestion
title: Messaging Service monitoring and management feature available as Public Preview
```

product_area: Platform services
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63290
version: 1021.73.0
---

> Added messaging management as beta feature (#8891) [GRAFT][release/cd] (#9003)

Reviewer 2: ```suggestion
It provides a public preview of a new feature: Messaging service - monitoring and management, which allows to monitor and manage the usage of messaging service. You can activate it in Administration using "Manage beta features" option in the user menu. Find out more about the feature in its user documentation: https://cumulocity.com/docs/standard-tenant/monitoring/#messaging-service
```

> It provides a public preview of a new feature: Messaging service - monitoring and management, which allows to monitor and manage the usage of messaging service. You can activate it in Administration using "Manage beta features" option in the user menu. Find out more about the feature in its user documentation: https://cumulocity.com/docs/standard-tenant/monitoring/#messaging-service

Reviewer: ```suggestion
The Messaging Service, the messaging component embedded in the {{< product-c8y-iot >}} platform, has been enhanced by a new monitoring and management feature which allows to monitor and manage the usage of the Messaging Service resources. This new feature is currently in public preview. You can activate it in the platform UI via the **Manage beta features** option in the user menu. Find out more about the feature in its [user documentation](/standard-tenant/monitoring/#messaging-service).
```

> The Messaging Service, the messaging component embedded in the {{< product-c8y-iot >}} platform, has been enhanced by a new monitoring and management feature which allows users to monitor and manage the usage of the Messaging Service resources. This new feature is currently in Public Preview status. You can activate it in the platform UI via the **Manage beta features** option in the user menu. Find out more about the feature in its [user documentation](/standard-tenant/monitoring/#messaging-service).

Reviewer: ```suggestion
The Messaging Service, the messaging component embedded in the {{< product-c8y-iot >}} platform, has been enhanced by a new monitoring and management feature which allows users to monitor and manage the usage of the Messaging Service resources. This new feature is currently in Public Preview. You can activate it in the platform UI via the **Manage preview features** option in the user menu. Find out more about the feature in its [user documentation](/standard-tenant/monitoring/#messaging-service).
```
